README.md

```diff
@@ -369,8 +388,82 @@ This section provides a step-by-step procedure for deploying the resource aggreg
 
      ```
      $ docker restart odim_1
-  
+      ```
     
-   **NOTE:**
-   To Refer to the ODIMRA API Guide, click on the github page below:  
-   https://odim-project.github.io/ODIM
+   
+   
+   
+   
+# Configuring proxy for Docker
+
+<blockquote>
+IMPORTANT:
+
+During the course of this procedure, you will be required to create files and copy content into them. In the content to be copied, substitute the parameters listed in the following table with their original values:
+
+|Parameter|Description|
+|---------|-----------|
+|`<Proxy_URL>` |Your company URL.|
+|`<ODIM_server_VM_IP>` |The IP address of the system where the resource aggregator is installed.|
+|`<FQDN>` |FQDN of the resource aggregator server.|
+
+</blockquote>
+
+### Procedure
+
+1.   In the home directory of odimra user, create a hidden directory called .docker, and then create a file called config.json. 
```

Comment:
please provide the command to create the hidden directory and mention that go to the .docker directory and then create a file config.json

$ mkdir .docker
$ cd .docker
$ vi config.json

Reply:
Added these commands.